cinema/migrations/0003_movie_duration.py

```diff
@@ -6,13 +6,13 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('cinema', '0002_initial'),
+        ("cinema", "0002_initial"),
     ]
 
     operations = [
         migrations.AddField(
-            model_name='movie',
-            name='duration',
+            model_name="movie",
+            name="duration",
             field=models.IntegerField(default=123),
             preserve_default=False,
```

Comment:
The `preserve_default` is set to `False`, which might cause issues with existing records. Consider setting `preserve_default=True` to ensure the default value is applied to existing records, as mentioned in the previous review.